src/silx/gui/data/DataViews.py

```diff
@@ -1302,8 +1335,27 @@ def createWidget(self, parent):
         widget.setOptionVisible(False)
         maskToolWidget = widget.getPlotWidget().getMaskToolsDockWidget().widget()
         maskToolWidget.setItemMaskUpdated(True)
+        self.__aggregationModeAction = AggregationModeAction(parent=widget)
+        widget.getPlotWidget().toolBar().addAction(self.__aggregationModeAction)
+        self.__aggregationModeAction.sigAggregationModeChanged.connect(self._aggregationModeChanged)
         return widget
 
+    def getAggregationModeAction(self) -> AggregationModeAction:
+        """Action toggling the aggregation mode action
+        """
+        return self.__aggregationModeAction
+
+    def _aggregationModeChanged(self):
+        plot = self.getWidget().getPlotWidget()
+        item = plot._getItem("image")
+
+        if item is None:
+            return
+
+        if isinstance(item, ImageDataAggregated):
+            aggregationMode = self.getAggregationModeAction().getAggregationMode()
+            item.setAggregationMode(aggregationMode)
+        
```

Comment:
Let's add the `AggregationModeAction` in the widget which controls the item in the plot.
- For `_Plot2dView` above, it is the widget that controls the plot item.
- Here, the widget controlling the plot item is the `StackView`, so best to control it's plot item from this widget. Otherwise it adds implicit relations: When looking at the `StackView` code, one can wonder why an `ImageDataAggregated item is used there.